### PR TITLE
Raise error if recordfile not set

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: integration_tests/suite/test_calls_recording.py [hunk 1]
```diff
@@ -106,6 +105,28 @@ def event_received():
...
+        )
+
+        variables = {'XIVO_CALLRECORDFILE': ''}
+        channel_id = self.given_call_not_stasis(variables=variables)
+
+        assert_that(
+            calling(self.calld_client.calls.start_record_from_user).with_args(channel_id),
+            raises(CalldError).matching(has_properties(status_code=403))
+        )
+
```
Comment (bloom1) [inline integration_tests/suite/test_calls_recording.py:129]: Zuul is dead, this probably does not pass tests

**Comment by bloom1** [comment]
recheck

File: integration_tests/suite/test_calls_recording.py (hunk 1 above)
Comment (bloom1) [reply integration_tests/suite/test_calls_recording.py:129]: Well it passes tests but I don't know why. Isn't the status_code supposed to be 500 in this case?

**Comment by fblackburn1** [comment]
recheck

**Pull request merged**